dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -442,6 +443,20 @@
 			dma-buf-size-alignment = <4>;
 			dma-copy-alignment = <4>;
 			status = "okay";
+			power-domain = <&io0_domain>;
+		};
+
+		lpgpdma2: dma@7e000 {
+			compatible = "intel,adsp-gpdma";
+			#dma-cells = <1>;
+			reg = <0x0007e000 0x1000>;
+			shim = <0x0007e800 0x1000>;
+			interrupts = <0x25 0 0>;
+			interrupt-parent = <&core_intc>;
```

Comment:
The previous version did not compile. Its weird that CI didn't catch it.

Reply:
I decided to move adding a third controller to a separate PR https://github.com/zephyrproject-rtos/zephyr/pull/53054